### DISCUSSION TITLE
slug in front matter has no effect over url

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#364] [Q&A] [answered]
Example: this document https://finec.mgimo.ru/finec-mgimo-v2/blog/%D0%BD%D0%B0%D1%88%D0%B8-%D0%B2%D1%8B%D0%BF%D1%83%D1%81%D0%BA%D0%BD%D0%B8%D0%BA%D0%B8-%D0%B8%D1%82%D0%BC%D0%B1-2021-%D0%B7%D0%B0%D1%89%D0%B8%D1%82%D0%B0-%D0%B2%D1%8B%D0%BF%D1%83%D1%81%D0%BA%D0%BD%D1%8B%D1%85-%D0%BA%D0%B2%D0%B0%D0%BB%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D1%85-%D1%80%D0%B0%D0%B1%D0%BE%D1%82/

has slug:

```
slug: "2021-itmb-graduates"
```

but slug has not effect over url.

**Comment by mroswell** [comment]
This may be a weird approach, but try creating a fake menu (that's not
linked to a theme), adding that to config; Put /2021-itmb-graduates in the
fake menu. I can't promise it'll work, but you could report back and let us
know.

Margie
--
http://HealthyCampaign.org
http://SugarWars.com
http://bit.ly/antiviral-nutrition

**Reply by epogrebnyak** [reply]
Thanks for the idea, it is a bit backwards indeed. slug in Hugo functionality - there is either a disable option or something else prevents form making it. I cannot create a menu for the blog item - this is not a path to go.

**Comment by epogrebnyak** [comment, marked as answer]
I needed:

```
[permalinks]
  blog = "/blog/:slug/"
```

in config.toml
